src/App/useInitializeCurrentUser.js

```diff
@@ -51,7 +49,6 @@ export const useInitializeCurrentUser = ({
     dexieCurrentUserInstance,
     isMermaidAuthenticated,
     isAppOnline,
-    isSyncInProgress,
     handleHttpResponseErrorWithLogoutAndSetServerNotReachableApplied,
   ])
 
```

Comment:
The current user was being initialized twice because this var in the dep array, which wasnt even used!!! I was hoping that might speed up tests (cause less rerendering), but no difference